.github/workflows/deploy-vercel-staging.yml

```diff
@@ -169,25 +169,81 @@ jobs:
             Commit ${{ github.sha }} successfully deployed to [${{ env.VERCEL_DEPLOYMENT_URL }}](${{ env.VERCEL_DEPLOYMENT_URL }})
             Deployment aliased as [${{ env.VERCEL_DEPLOYMENT_AUTO_BRANCH_ALIAS }}](https://${{ env.VERCEL_DEPLOYMENT_AUTO_BRANCH_ALIAS }})
 
+  await-for-vercel-deployment:
+    name: Await this deployment to be ready (Ubuntu 18.04)
+    runs-on: ubuntu-18.04
+    needs: start-staging-deployment
+    steps:
+      - name: Resolving deployment url from Vercel
```

Comment:
We're gonna have a lot of conflicts with https://github.com/UnlyEd/next-right-now/pull/225, I'd rather merge https://github.com/UnlyEd/next-right-now/pull/225 first and adapt this PR then, because https://github.com/UnlyEd/next-right-now/pull/225 is more complicated to debug.